app/src/main/java/com/kickstarter/libs/utils/KoalaUtils.java

```diff
@@ -101,33 +106,27 @@ private KoalaUtils() {}
     return MapUtils.prefixKeys(properties, prefix);
   }
 
-  public static @NonNull Map<String, Object> projectProperties(final @NonNull Project project) {
-    return projectProperties(project, "project_");
-  }
-
-  public static @NonNull Map<String, Object> projectProperties(final @NonNull Project project, final @NonNull String prefix) {
-    return projectProperties(project, null, prefix);
+  public static @NonNull Map<String, Object> projectProperties(final @NonNull Project project, final @Nullable User loggedInUser) {
+    return projectProperties(project, loggedInUser, "project_");
   }
 
   public static @NonNull Map<String, Object> projectProperties(final @NonNull Project project, final @Nullable User loggedInUser, final @NonNull String prefix) {
     final Map<String, Object> properties = new HashMap<String, Object>() {
       {
         put("backers_count", project.backersCount());
+        put("comments_count", project.commentsCount());
         put("country", project.country());
+        put("duration", Math.round(ProjectUtils.timeInSecondsOfDuration(project)));
         put("currency", project.currency());
         put("goal", project.goal());
-        put("pid", project.id());
+        put("has_video", project.video() != null);
+        put("hours_remaining", (int) Math.ceil(ProjectUtils.timeInSecondsUntilDeadline(project) / 60.0f / 60.0f));
         put("name", project.name());
```

Comment:
What do you think about adding a method in `ProjectUtils` to do these calculations for `hours_remaining` and `percent_raised`?

Reply:
-hisses- no. It's only used here. 😛